apps/prairielearn/python/prairielearn.py

```diff
@@ -464,11 +464,11 @@ def inner_html(element: lxml.html.HtmlElement) -> str:
     return inner
 
 
-def compat_get(object, attrib, default):
-    if attrib in object:
-        return object[attrib]
+def compat_get(obj, attrib, default):
```

Comment:
This needs to be checked to ensure there are no downstream callers with `obj=`.

Reply:
I don't see any usages of this. Maybe we can remove it entirely.

Reply:
Sounds good! Removed in 5a061c270.